<launch>
	<!--<machine name="telecomando" address="10.142.0.111" user="ubuntu" />

	<node machine="telecomando" name="rosserial" pkg="rosserial_arduino" type="serial_node.py" output="screen">
		<param name="port"              type="string"               value="/dev/ttyACM0"/>
		<param name="baud"              type="int"                  value="57600"/>

	<node name="communication" pkg="reseq_ros" type="communication.py" output="screen"/>
	</node>-->
	
	<node name="agevar_in" pkg="reseq_ros" type="agevar_in.py" output="screen"/>
	<node name="agevar_core" pkg="reseq_ros" type="agevar_core.py" output="screen"/>
	<node name="agevar_out" pkg="reseq_ros" type="agevar_out.py" output="screen"/>

	<!--
	
	<node name="rqt_gui" pkg="rqt_gui" type="rqt_gui"/>
	
	-->

</launch>
